### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by JD-113**
@ShutongChen9 Review variable and function names and  ensure consistent code formatting and style

### CODE REVIEW

File: prac_05/word_occurrences.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:   35  minutes
+"""
+
+WORD_TO_COUNT = {}
```
Comment (awen126) [inline prac_05/word_occurrences.py]: Lowercase maybe better.

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        affirmation = input(f"Is your name {name}? (Y/n)")
+        if affirmation.upper() != "Y" and affirmation.upper() != "":
+            name = input("Name: ")
+        email_to_name[email] = name
+        email = input('Email: ')
+
+    for email, name in email_to_name.items():
+        print(name, email)
+
+    pass
```
Comment (awen126) [inline prac_05/emails.py]: "pass" can be delete.

File: prac_05/word_occurrences.py (hunk 1 above)
Comment (JD-113) [reply prac_05/word_occurrences.py]: Ok yea i think lower is better here thanks

File: prac_05/emails.py (hunk 2 above)
Comment (JD-113) [reply prac_05/emails.py]: Thanks for picking this this up was overlooked

**Pull request merged**